CHANGES.md

```diff
@@ -23,3 +47,20 @@ Introducing linkerd-tcp. :balloon:
 * Run a TCP load balancer for the [Linkerd][https://linkerd.io] service mesh.
 * Support TLS and integrate with Namerd for service discovery.
 * Read a YAML or JSON configuration file at startup.
+
+
++    available: tacho::Gauge,
++    failed: tacho::Gauge,
++    retired: tacho::Gauge,
++    pending: tacho::Gauge,
++    open: tacho::Gauge,
++    waiters: tacho::Gauge,
++    poll_time: tacho::Timer,
++    attempts: tacho::Counter,
++    unavailable: tacho::Counter,
++    connects: tacho::Counter,
++    timeouts: tacho::Counter,
++    refused: tacho::Counter,
++    failures: tacho::Counter,
++    connect_latency: tacho::Timer,
++    connection_duration: tacho::Timer,
```

Comment:
???

Reply:
agh, I copied and pasted in some code while i was trying to manually diff the metrics and forgot to delete it >.<